sympy/utilities/tests/test_lambdify.py

```diff
@@ -581,3 +582,9 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+def test_10375():
+    import numpy as np
+    m = Min(x, y, z)
+    l = lambdify([x,y,z], m)
```

Comment:
I modified NumPyPrinter in gh-10442, I think it's cleaner than `isinstance` checks unless I've overlooked something.